sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/ballerina-4.html

```diff
@@ -0,0 +1,36 @@
+<p>
+    Class methods can be isolated. An isolated method is same as an isolated function with self treated as a parameter.
```

Comment:
Same comments re: phrasing in terms of the problem.

Reply:
Check others too.

Reply:
Fixed with ce8f86feeeaf9d99bdb56f0f07c4878660cb0af1